src/index.html

```diff
@@ -99,10 +320,52 @@ <h2 class="quiz-card__heading">Question</h2>
         </ul>
       </section>
     </main>
-
+    <!-- create -->
+    <main class="page-create hidden">
+      <form method="post" class="create-form">
+        <div class="create-form__input-group">
+          <label for="question" class="create-form__heading">Question</label>
```

Comment:
This is not a heading but a label
```suggestion
          <label for="question" class="create-form__label">Question</label>
```